src/main/java/io/github/rabobank/shadow_tool/DefaultEncryptionService.java

```diff
@@ -3,19 +3,27 @@
 import org.bouncycastle.util.encoders.Base64;
 
 import javax.crypto.Cipher;
+import java.security.GeneralSecurityException;
 
-class DefaultEncryptionService implements EncryptionService {
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * The default Encryption Service used by Shadow Tool. It uses a {@link Cipher} to encrypt values.
+ *
+ * @see Cipher
+ */
+public class DefaultEncryptionService implements EncryptionService {
     private final Cipher cipher;
 
-    DefaultEncryptionService(final Cipher cipher) {
+    public DefaultEncryptionService(final Cipher cipher) {
         this.cipher = cipher;
     }
 
     @Override
     public String encrypt(final String value) {
         try {
-            return Base64.toBase64String(cipher.doFinal(value.getBytes()));
-        } catch (final Exception e) {
+            return Base64.toBase64String(cipher.doFinal(value.getBytes(UTF_8)));
```

Comment:
The input arg (`value`) of this method can be `null` or `empty`. Likely we should handle it properly

Reply:
I'd say that's a non-issue. Yes, technically when you call this manually then this could be null. But, from the Shadow Tool this will never be called with a `null` value.